README.md

```diff
@@ -279,7 +279,45 @@ To use the WAF service, use the API Token provided to you.
 An access rule identifies legitimate traffic and threats by ASN, Cookie, Country, 
 IP Address, Referrer, URL, User agent, HTTP method, Media type, File extension, 
 and Request headers.
-```
+
+For detailed information about Access Rules in WAF, please read the [official documentation](https://docs.edgecast.com/cdn/#Web-Security/Access-Rules.htm).
+
+#### Bot Rule Sets
+Use bot rules to require a client (e.g., a web browser) to solve a challenge 
+before resolving the request. WAF blocks traffic when the client cannot solve 
+this challenge within a few seconds. Basic bots typically cannot solve this type
+of challenge and therefore their traffic is blocked. This prevents them from 
```

Comment:
Basic bots typically cannot solve this type
of challenge, and therefore their traffic is blocked.

Reply:
Updated